packages/components/react/src/text/__tests__/text.visual.test.tsx

```
@@ -0,0 +1,208 @@
+import { render } from "@testing-library/react";
```

ESLint

[error] Unable to resolve path to module '@testing-library/react'.

(import-x/no-unresolved)